agenta-backend/agenta_backend/models/db_models.py

```diff
@@ -6,6 +6,7 @@
 class OrganizationDB(Model):
     name: str = Field(default="agenta")
     description: str = Field(default="")
+    members: Dict[str, str]
```

Comment:
Same thing, we should reference ids not use str. Please read odomantic reference to see how we can achieve that

Reply:
the first str here represents the user id.